app/decorators/paginating_decorator.rb

```diff
@@ -0,0 +1,3 @@
+class PaginatingDecorator < Draper::CollectionDecorator
+  delegate :current_page, :total_pages, :limit_value
+end
```

Comment:
please add the newline at the end of each file. This can be configured in your text editor.